joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -432,6 +446,9 @@ rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
 JointTrajectoryController::on_deactivate(const rclcpp_lifecycle::State & previous_state)
```

Comment:
Do we also need `release_interfaces` method? I can not find how CM is using it.

Reply:
Yeah I think you're right. Or it should be done in the controller manager after deactivating, but for now I'll add it here.